internal/services/excel_service.go

```
@@ -0,0 +1,186 @@
+            Name:     strings.TrimSpace(name),
+            NameEn:   "",
+            Price:    0.00,
+        })
+    }
+    return menuItems
+}
+
+// 식사 타입별 설정 반환
+func (s *ExcelService) getMealTypeConfigs() []models.MealTypeConfig {
+    return []models.MealTypeConfig{
+        {"Breakfast", 7, 16},
```

golangci-lint (1.64.8)

composites: github.com/School-meal-lover/backend/internal/models.MealTypeConfig struct literal uses unkeyed fields

(govet)